app/scripts/editor/token_highlighter.js

```diff
@@ -0,0 +1,108 @@
+/* global ace */
+'use strict';
+
+var Range = ace.require('ace/range').Range;
```

Comment:
I tried it by building and running it through dist and it didn't throw an error. This is because 'range' is defined inside ace.js itself.